raspi/access_point/register.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import sys
+import time
+
+import requests
+import json_config
+
+def get_token(login,password):
+    url = 'http://api.susi.ai/aaa/login.json?type=access-token'
+    PARAMS = {
+        'login':login,
+        'password':password,
+    }
+    r1 = requests.get(url, params=PARAMS).json()
+    return r1['access_token']
+
+def device_register(access_token):
+    url='https://api.susi.ai/aaa/addNewDevice.json?&macid=21:12:23:34:12:23&name=meow&room=home&latitude=12.1&longitude=23.3'
```

Comment:
Why these strange values for latitude and longitude? The GPS position should be the current one.

Also the MAC ID cannot be fixed!

Reply:
@norbusan I'm still working on this 
I'll mark this ready once done